scripts/conda/development_setup_with_conda.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+cd "$(dirname "$0")"
+
+set -e
+set -x
+
+if ! command -v mamba &> /dev/null
+then
+    curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+    bash Miniforge3-$(uname)-$(uname -m).sh -b
+    mamba config --set auto_activate_base false
```

Comment:
I think there is a cleaner way to do this, I will test this more closely.

Reply:
First of all -- apologies for not testing this code in the first place. I thought it would require me to uninstall conda completely, which would be a hassle, but I simply needed to remove it from my PATH. I have addressed this in a different way, the one recommended by the Miniforge docs, and tested that it works now.